backend/functions/src/student/functions.js

```diff
@@ -65,6 +65,7 @@ const addStudentSurveyResponse = async (
       year,
       preferredWorkingTime,
       groups: studentCrses,
+      time: admin.firestore.FieldValue.serverTimestamp(),
```

Comment:
Agree with @mluo24 about the naming of this, now that other time-related properties are called `createTime` and `updateTime`. Maybe `timestamp` to make it more clear (since there's a `year` property for students too) or `surveyTime` or `submissionTime` to parallel how it's worded for groups.